### ISSUE TITLE
[FEAT] Custom Error Page 404 

### ISSUE THREAD

**Issue by virajchandra51** [AKD-01/blogweet#72]
I will design an Error Page for calls or redirecting to pages which do not exist.

Hello!
I am Viraj Chandra, a proud contributor at GSSoC '23. I would like to the contribute to this project by designing an Error Page for calls or redirecting to pages which do not exist.

Things I will be doing as of now:
- [ ]  Designing the error page 404

@AKD-01 Please Assign me this.

**Comment by AKD-01** [comment]
Hi, @virajchandra51 someone is already working on this issue. So, you may wait before starting with this. I will update you soon.

Can raise another new issue as well.

**Comment by aniketnimkar** [comment]
I'm eager to make a valuable contribution to this issue as part of the GSSOC 2023.

**Comment by AKD-01** [comment]
Blogweet now has a custom error page.  So closing this issue.
